Documentation/ceph-cluster-crd.md

```diff
@@ -96,6 +98,16 @@ log enough to ignore network blips where mons are failed over too often.
 - `ROOK_MON_HEALTHCHECK_INTERVAL`: The frequency with which to check if mons are in quorum (default is 45 seconds)
 - `ROOK_MON_OUT_TIMEOUT`: The interval to wait before marking a mon as "out" and starting a new mon to replace it in the quroum (default is 600 seconds)
 
+**volumeClaimTemplate**. This field is optional, and when not provided, HostPath
+volume mounts are used.  The current set of fields from template that are used
+are `storageClassName` and the `storage` resource request and limit. The default
+storage size request for new PVCs is `10Gi`. Ensure that associated storage
+class is configured to use `volumeBindingMode: WaitForFirstConsumer`. This
+setting only applies to new monitors that are created when the requested number
+of monitors increases, or when a monitor fails and is recreated. Log and crash
```

Comment:
i don't think you need to mention the log and crash data here. That detail can stay in the design doc.